CHANGELOG.md

```diff
@@ -2,9 +2,13 @@
 
 ## [7.3.0] (IN PROGRESS)
 
-* Clear assigned users redux when leaving Assigned Users page. (UIEH-1315)
 * Hide filter by Tag when user doesn't have related permissions. (UIEH-1313)
```

Comment:
The `UIEH-1313` is already in the `master` branch. Should it be in 7.3.0?

Reply:
I only cherry-picked `UIEH-1169` and `UIEH-1315` to `b7.2` branch and tagged release version there, so `UIEH-1313` won't be a part of `7.2.1`